sharedUtils/prepareGranuleAccessParams.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Takes granule params an returns the params to be used in a CMR request
+ * @param {Object} params Queue messages from SQS
+ * @returns {Object} The granule params
+ */
+export const prepareGranuleAccessParams = (params = {}) => {
```

Comment:
Seems like you could just return instead of overwriting the granule params. If `conceptIdsFromParams` then return an object, otherwise just return `params`.